pages/hosted_private_cloud/nutanix_on_ovhcloud/80-first-steps/guide.fr-fr.md

```diff
@@ -0,0 +1,496 @@
+
+Cliquez ensuite sur l'onglet `Frontends`{.action} et sur `Ajouter un frontend`{.action}.
+
+![Configure Load Balancer 06-ssh](images/config-lb6-ssh.PNG){.thumbnail}
+
+Nommez votre frontend, choisissez le protocole `TCP`{.action} et modifiez ces valeurs :
+
+- **Port** : `22`
+- **Datacenter** : `ALL`
+- **Ferme par défaut** : `rebond-ssh (TCP)`
+
+Cliquez sur `Ajouter`{.action}.
+
+![Configure Load Balancer 07 ssh](images/config-lb7-ssh.PNG){.thumbnail}
+
+Cliquez sur `Appliquer la configuration`{.action}.
+
+![Configure Load Balancer 08 ssh](images/03a-configureloadbalancer07.png){.thumbnail}
+
+Sélectionnez le `Datacenter`{.action} et cliquez sur `Appliquer la configuration`{.action}.
+
+![Configure Load Balancer 09 ssh](images/03a-configureloadbalancer08.png){.thumbnail}
+
+Dans l'onglet `Tâches`{.action}, vous pourrez suivre l'avancement de l'application des changements.
+
+![Configure Load Balancer 10 ssh](images/03a-configureloadbalancer09.png){.thumbnail}
+
+Le Load Balancer est désormais configuré. Utilisez votre terminal préféré pour vous connecter sur votre machine.
+Vous pouvez saisir le nom DNS de votre cluster ou l'IP du Load Balancer :
+
+![Connect to CVM](images/cvm-ssh-linux2.PNG){.thumbnail}
+
+Vous pouvez désormais rebondir sur les différents éléments du cluster :
+
+- CVMs
+- Hôtes
+- Prism Element
+
+Par exemple, pour une cvm, saisissez dans le terminal :
+
+```bash
+ssh nutanix@ipcvm
+```
+
+![Connect to CVM](images/cvm-ssh-linux.PNG){.thumbnail}
+
+#### Rebond via RDP
+
+##### Création et configuration de la VM Windows
+
+Importez votre image Windows dans le cluster. Consultez notre guide sur l'[importation d'images dans Nutanix](/pages/hosted_private_cloud/nutanix_on_ovhcloud/05-image-import).
+
+Dans le menu de gauche de **Prism Central**, dépliez `Compute & Storage`{.action} et cliquez sur `VMs`{.action}.
+
+![Tableau de Bord Prism Central - Menu VMs](images/PrismCentralDashBooardWithVMMenu.PNG){.thumbnail}
+
+Cliquez sur le bouton `Create VM`{.action}.
+
+![Tableau de Bord Prism Central - Gestion des VMs ](images/PrismCentralDashVmDashBoard.PNG){.thumbnail}
+
+Saisissez un nom dans `Name`{.action}, choisissez les options dans `VM Properties`{.action} et cliquez sur `Next`{.action}.
+
+![Création d'une machine virtuelle - Etape 1](images/CreateVM01.PNG){.thumbnail}
+
+###### Ajout d'un disque système
+
+Cliquez sur le bouton`Attach Disk`{.action}.
+
+![Création d'une machine virtuelle - Etape 2](images/CreateVM02.PNG){.thumbnail}
+
+Saisissez **60** dans le champ `capacity` et cliquez sur `Save`{.action} pour créer un disque de 60 Go.
+
+![Création d'une machine virtuelle - Etape 3](images/CreateVM03.PNG){.thumbnail}
+
+###### Ajout de l'image ISO de l'installation de Windows Server 2022
+
+L'image doit être importée avant d'être utilisable dans une nouvelle machine virtuelle.
+
+Pour plus de détails sur l'importation d'images, reportez-vous à la section « [Aller plus loin](#gofurther) » de ce guide.
```

Comment:
There is no information about this in the gofurther part. Should we include here a link to the images importation guide (/pages/hosted_private_cloud/nutanix_on_ovhcloud/05-image-import) ?